plugins/org.genivi.faracon/src/org/genivi/faracon/franca2ara/ARAPrimitveTypesCreator.xtend

```diff
@@ -16,18 +16,16 @@ class ARAPrimitveTypesCreator extends Franca2ARABase {
 
 	private static final Logger logger = Logger.getLogger(ARAPrimitveTypesCreator.name)
 	
-	//todo: use a more reliable path
-//	val static String PATH_TO_STD_ARXML_FILES = "../../plugins/org.genivi.faracon/models/stdtypes.arxml"
-	val static String PATH_TO_STD_ARXML_FILES = "C:/Users/tgoerg/git/franca_ara_tools/plugins/org.genivi.faracon/models/stdtypes.arxml"
+	val static String PATH_TO_STD_ARXML_FILE = "stdtypes.arxml"
```

Comment:
This is defined also in ARAConnector.java. One of these redundant definitions should probably be avoided.

Reply:
This problem goes a little bit deeper. At both locations the ARXML that contains definitions of standard primitive types is loaded although these definitons are generally necessary for each AUTOSAR model (for boch directions, loading and storing). A better solution might be a special resource set class for AUTOSAR models which implicitly loads the standard defintions.

Reply:
So maybe we should tackle this at a later stage (we planned to build a configurable solution for AUTOSAR stdtypes anyway, e.g. using an extension mechanism). So probably keep it for now and create another issue so we won't forget this?

Reply:
I have just solved the problem by extracting the handling of standard primitive types into a specialized resource set class, as suggested above. Furthermore, this new resource set class will be a proper place to add configurability for AUTOSAR standard types at a later stage.